src/general/MainMenu.tscn

```diff
@@ -1019,51 +973,14 @@ rect_min_size = Vector2( 400, 350 )
 WindowTitle = "MOD_LOAD_ERRORS"
 ErrorMessage = "MOD_LOAD_ERRORS_OCCURRED"
 
-[node name="ThanksForBuyingDialog" parent="." instance=ExtResource( 42 )]
-anchor_right = 1.0
-anchor_bottom = 1.0
-popup_exclusive = true
-WindowTitle = "THANK_YOU_TITLE"
-
-[node name="VBoxContainer" type="VBoxContainer" parent="ThanksForBuyingDialog"]
-margin_right = 500.0
-margin_bottom = 192.0
+[node name="ThanksForBuyingDialog" parent="." instance=ExtResource( 59 )]
+margin_right = 163.0
 rect_min_size = Vector2( 500, 0 )
-size_flags_horizontal = 3
-
-[node name="ThanksText" parent="ThanksForBuyingDialog/VBoxContainer" instance=ExtResource( 55 )]
-margin_right = 500.0
-ExtendedBbcode = "THANKS_FOR_BUYING_THRIVE"
-
-[node name="Spacer" type="Control" parent="ThanksForBuyingDialog/VBoxContainer"]
-margin_top = 26.0
-margin_right = 500.0
-margin_bottom = 135.0
-rect_min_size = Vector2( 0, 5 )
-mouse_filter = 2
-size_flags_vertical = 3
-
-[node name="CheckBox" parent="ThanksForBuyingDialog/VBoxContainer" instance=ExtResource( 54 )]
-margin_top = 139.0
-margin_right = 500.0
-margin_bottom = 156.0
-custom_fonts/font = ExtResource( 51 )
-pressed = true
-text = "DISMISS_PERMANENTLY"
-
-[node name="HBoxContainer" type="HBoxContainer" parent="ThanksForBuyingDialog/VBoxContainer"]
-margin_top = 160.0
-margin_right = 500.0
-margin_bottom = 192.0
-size_flags_horizontal = 3
-
-[node name="Button" type="Button" parent="ThanksForBuyingDialog/VBoxContainer/HBoxContainer"]
-margin_left = 231.0
-margin_right = 268.0
-margin_bottom = 32.0
-size_flags_horizontal = 6
-custom_fonts/font = ExtResource( 53 )
-text = "OK"
+WindowTitle = "THANK_YOU_TITLE"
+HideCancelButton = true
+DialogText = "THANKS_FOR_BUYING_THRIVE"
+CenterText = false
+NoticeType = 1
```

Comment:
Here is the case where dismissing a warning is completely incorrect. We should not warn the user of the fact that they bought the game and support us...